lib/GeoPointCaster.utils.js

```diff
@@ -106,11 +105,12 @@ function getArraySchema(field) {
 
 function getObjectSchema(field) {
   if ('schema' in field && field.schema) {
-    return { ...field.schema, type: 'object' }
+    return { ...field.schema, ...getMiaConfiguration(field), type: 'object' }
   }
 
   return {
     type: 'object',
+    ...getMiaConfiguration(field),
   }
```

Comment:
I'd rather be consistent with the order of the keys and leave `type: 'object'` at the end.

```suggestion
  return {
    ...getMiaConfiguration(field),
    type: 'object'
  }
```